emmarin/cl/cl/src/cl/pact.rs

```diff
@@ -0,0 +1,37 @@
+use crate::cl::{PartialTx, PartialTxWitness};
+use crate::zone_layer::notes::ZoneId;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+pub struct Pact {
+    pub tx: PartialTx,
+    pub to: Vec<ZoneId>,
```

Comment:
why vec? do we support transfers to multiple zones in one PACT?

Reply:
ah, got it, corresponds with the tx outputs.

Reply:
> why vec? do we support transfers to multiple zones in one PACT?

yes, why not?

Reply:
yeah makes sense